challenges/prototypes.js

```diff
@@ -5,29 +5,41 @@
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
```

Comment:
Well done, however, you do not need to wrap your numbers in string literals... you could just do something like return this.length * this.width * this.height without a literal template. 19/20 due to newObj was request to be named cuboid